devscripts/prepare_manpage.py

```diff
@@ -43,6 +49,33 @@ def main():
         outf.write(readme)
 
 
+def filter_excluded_sections(readme):
+    EXCLUDED_SECTION_BEGIN_STRING = '<!-- MANPAGE: BEGIN EXCLUDED SECTION -->'
+    EXCLUDED_SECTION_END_STRING = '<!-- MANPAGE: END EXCLUDED SECTION -->'
+    return re.sub(
+        '(?s)%s.+?%s\n' % (EXCLUDED_SECTION_BEGIN_STRING, EXCLUDED_SECTION_END_STRING),
```

Comment:
```suggestion
        rf'(?s){re.escape(EXCLUDED_SECTION_BEGIN_STRING)}.+?{re.escape(EXCLUDED_SECTION_END_STRING)}\n'
```
do `re.escape` when putting literal string inside regex